<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inserisci_Movimento_UI</class>
 <widget class="QWidget" name="Inserisci_Movimento_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Inserisci Movimento</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(254, 235, 156);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_InsDip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Inserisci movimento</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-radius: 14px;
Width:28px;
Height:28px;
background-color: rgb(55, 92, 108);
color: rgb(255, 255, 255);
border-style: outset;



}

QPushButton:hover{

background-color: rgb(92, 154, 181);
}
QPushButton:pressed{
border-style: inset;
background-color: rgb(48, 80, 91);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_DescrizioneVoce">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>Descrizione voce</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_DescrizionVoce">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;

padding: 3px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_TipoSegno">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>               Tipo </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Segno">
       <property name="styleSheet">
        <string notr="true">QComboBox{
background-color: rgb(255, 255, 255);
border-radius:10px;
padding: 5px;
}


QFontComboBox::down-arrow {
 rgb(0,0,0);
image: url(C:/Users/matteo.straccia/Desktop/Uni/Ingegneria del softwere/Progetto/UI/Immagini/MicrosoftTeams-image.png);
}



QComboBox::drop-down{
border-radius:10px;
padding: 5px;
}



QComboBox::drop-down:hover{
background-color: rgb(92, 176, 150);
border-radius:5px;
padding: 5px;
}



QComboBox::down-arrow:on {
top: 1px;
}</string>
       </property>
       <item>
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Importo">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>       Importo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Importo">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;

padding: 3px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Euro">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(92, 164, 144);</string>
       </property>
       <property name="text">
        <string>â‚¬ </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_InserisciMov">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton{
border-radius: 15px;
	background-color: rgb(55, 92, 108);
padding: 10px;
color: rgb(255, 255, 255);
border-style: outset;
Width:100px;}

QPushButton:hover{
background-color:rgb(92,154,181);}  </string>
       </property>
       <property name="text">
        <string>Inserisci</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Immagini/plus-304947_1280.png</normaloff>Immagini/plus-304947_1280.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
